src/MoBi.Core/Domain/Services/CloneManagerForSimulation.cs

```diff
@@ -30,7 +30,8 @@ public IMoBiSimulation CloneSimulation(IMoBiSimulation simulationToClone)
       {
          var model = _cloneManagerForModel.CloneModel(simulationToClone.Model);
 
-         var simulation = _simulationFactory.CreateFrom(CloneSimulationConfiguration(simulationToClone.Configuration), model);
+         //TODO
+         var simulation = _simulationFactory.CreateFrom(CloneSimulationConfiguration(simulationToClone.Configuration), model, null);
```

Comment:
that's a serious issue here. We have the model and in the original simulation, the `EntitySource` is pointing to id of the model. Now we are cloning the configuration and cloning also the model. What should be the source then?
Do we even care of the configuration in this case? Maybe the only thing that matter was the ORIGINAL configuration used to create the first simulation before clons (e.g. pointing to real BB)
For sure what will be off is the EntitySources objects. It has id in the model that is being cloned. I think we might have to save by path instead in the simulation. Thoughts?